src/modules/projects/ui/views/project-view.tsx

```
@@ -37,8 +43,49 @@ const ProjectView = ({ projectId }: ProjectViewProps) => {
+            <TabsContent value="preview">
+              {!!activeFragment ? (
+                <FragmentWeb data={activeFragment} />
+              ) : (
+                <div className="flex h-full items-center justify-center text-muted-foreground">
+                  <LucidePlus className="w-4 h-4 mr-2" />
+                  <span>Select a fragment to preview</span>
+                </div>
+              )}
+            </TabsContent>
+            <TabsContent value="code" className="flex-1 min-h-0">
+              {!!activeFragment?.files ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)